src/ansible_navigator/actions/quit.py

```diff
@@ -21,12 +21,10 @@ def __init__(self, args: ApplicationConfiguration):
 
     # pylint: disable=unused-argument
     def run(self, interaction: Interaction, app: AppPublic) -> Interaction:
-        """Handle :doc
+        """Handle a request to quit the application.
```

Comment:
It feels like this does not explain what this method does. I look into the body and only see a logging invocation. Why does not the docstring say that this method doesn't handle anything but just logs the request, for example?

Reply:
the comment below the docstring has been incorporated into the docstring now which I think might explain it better

Reply:
Yes, it mentions the activity delegation but fails to mention that this method does not delegate or do anything functional except for logging the intercepted request. Am I missing anything?

Reply:
updated mo

Reply:
Resolved.